src/OmniSharp.Cake/Services/RequestHandlers/Navigation/FindSymbolsHandler.cs

```diff
@@ -21,12 +21,18 @@ public FindSymbolsHandler(OmniSharpWorkspace workspace)
 
         public override Task<QuickFixResponse> Handle(FindSymbolsRequest request)
         {
+            if (request != null && request.Filter != null && request.Filter.Length < request.MinFilterLength)
```

Comment:
This changes how the API works, so we probably need to implement this as a V2 API